src/styles/block/_calendar.scss

```diff
@@ -0,0 +1,61 @@
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  &__list {
+    display: flex;
+    flex-wrap: wrap;
+    width: 7 * $block-width + 10px;
```

Comment:
where does `10px` come from?

Reply:
Use flex with 1px gap and limit its width to exactly 7 columns + 10px paddings